metadata.json

```
@@ -14,11 +14,11 @@
     },
     {
       "name": "puppet-archive",
-      "version_requirement": ">= 4.0.0 < 6.0.0"
+      "version_requirement": ">= 4.0.0 < 6.0.2"
```

I highly recommend to pin to the next major version:
```suggestion
      "version_requirement": ">= 4.0.0 < 7.0.0"
```

that's the current best practice. also Vox Pupuli does semantic versioning. if module 6.0.0 works for you, you can assume that all 6.x releases will work for you.

---

Thanks for the catch we  just changed it